### PR TITLE
[lldb] Add script to copy the static bindings

### PR DESCRIPTION
**Pull request by JDevlieghere**

### CODE REVIEW

**Comment by adrian-prantl** [comment]
What do you think about adding a comment or usage message that explains when to run this script / what is used for?

**Comment by JDevlieghere** [comment]
> What do you think about adding a comment or usage message that explains when to run this script / what is used for?

Sure, sounds good!

**Review by medismailben — APPROVED** [review]
LGTM taking into account Adrian's comment.

File: lldb/scripts/copy-static-bindings.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+"""SWIG Static Binding Copier
+
+This script copies over the Python bindings generated by SWIG from the build
+directory to the source directory. Before using the script, make sure you have
+LLDB_USE_STATIC_BINDINGS set to OFF by looking at CMakeCache.txt in the LLDB
```
Comment (medismailben) [inline lldb/scripts/copy-static-bindings.py:7]: Nit: I guess we could check that and exit early with an error message to change CMake option ?

Comment (JDevlieghere) [reply lldb/scripts/copy-static-bindings.py:7]: I considered doing that, but there are many values that CMake accepts, such as `On`, `on`, `1`, etc. It didn't seem worth maintaining a shadow implementation for that.

Comment (medismailben) [reply lldb/scripts/copy-static-bindings.py:7]: Fair 💯

**Comment by JDevlieghere** [comment]
@swift-ci please test

**Comment by JDevlieghere** [comment]
@swift-ci please test macos

**Review by adrian-prantl — APPROVED** [review]

**Pull request merged**